### PR TITLE
Add load test to the udp module

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/2117
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [x] Added tests

### CODE REVIEW

**Review by dilanSachi — APPROVED** [review]
LGTM

File: load-tests/file_server/src/main.bal [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        return false;
+    }
+}
+
+public function publishMessages() returns error? {
+    udp:ConnectClient socketClient = check new ("localhost", PORT, timeout = 1);
+    startedTime = time:utcNow();
+    int endingTimeInSecs = startedTime[0] + 200;
+    byte sequenceNo = 0;
+    while time:utcNow()[0] <= endingTimeInSecs {
```
Comment (ayeshLK) [inline load-tests/file_server/src/main.bal:66]: We could refactor this as follows:
```ballerina
byte sequenceNo = 0;
time:Utc expiryTime = time:utcAddSeconds(startedTime, 200);
while time:utcDiffSeconds(expiryTime, time:utcNow()) > 0D {
  // logic here
}
```

File: load-tests/file_server/src/main.bal [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            };
+        }
+        return false;
+    }
+}
+
+public function publishMessages() returns error? {
+    udp:ConnectClient socketClient = check new ("localhost", PORT, timeout = 1);
+    startedTime = time:utcNow();
+    int endingTimeInSecs = startedTime[0] + 200;
```
Comment (ayeshLK) [inline load-tests/file_server/src/main.bal:64]: Here we have only added `200 seconds`. I think this number should be changed to `3600` if we need to run this load test for an hour :thinking:

Comment (Bhashinee) [reply load-tests/file_server/src/main.bal:64]: Yes, Planning to update the time after a test run.

File: load-tests/file_server/src/main.bal (hunk 1 above)
Comment (Bhashinee) [reply load-tests/file_server/src/main.bal]: Fixed with [1a4259f](https://github.com/ballerina-platform/module-ballerina-udp/pull/394/commits/1a4259fbf6aff42f6aa32fc4e64db7dd96e5f304)

**Pull request merged**